include/RAJA/policy/sycl.hpp

```diff
@@ -29,6 +29,7 @@
 #include "RAJA/policy/sycl/forall.hpp"
 #include "RAJA/policy/sycl/policy.hpp"
 #include "RAJA/policy/sycl/reduce.hpp"
+#include "RAJA/policy/sycl/multi_reduce.hpp"
```

Comment:
I don't see this file anywhere.

Reply:
You are correct, that is there to remind me to add that implementation.